apps/cookbook/src/app/shared/api-description/api-description.shared.scss

```
@@ -0,0 +1,32 @@
+@import './libs/designsystem/src/lib/scss/utils';
+
+.api-description {
+  margin-bottom: size('xl');
+
+  border-spacing: 0;
+  border-collapse: collapse;
+  width: 100%;
+
+  thead {
+    background-color: get-color('light-tint');
+  }
+
+  th {
+    font-weight: font-weight('normal');
+    border: none;
+    text-align: left;
+    height: size('xl');
+    border-bottom: none;
+  }
+
+  td,
+  th {
+    border: 1px solid get-color('medium');
+    border-width: 1px 1px 1px 1px;
```

Consider simplifying:
```suggestion
    border-width: 1px;
```